style.css

```diff
@@ -0,0 +1,740 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&family=Roboto:ital,wght@1,100;1,400&display=swap");
```

Comment:
Please make sure your project follows the design guidelines as specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=You%20should%20follow,changes%20are%20acceptable.). By this I mean using the provided font-face (weight, color, family, etc), margins, padding, alignments, and others. 


**The provided design**
![Screenshot from 2022-10-26 11-59-53](https://user-images.githubusercontent.com/92395484/198090524-29b5a786-431c-493f-9177-096777cbbc91.png)

**Your Project**
![image](https://user-images.githubusercontent.com/92395484/198090660-f5da736a-7d21-4528-8a84-be4e6ff51adb.png)

Make sure your mobile version is responsive regarding font-sizes :heavy_check_mark: 



**Note:** In this case you are encouraged to use your own images and content :smile: :+1: